### PR TITLE
more porting

### PR DESCRIPTION
**Pull request by Machine-Maker**

### CODE REVIEW

File: src/main/resources/templates/projects/discuss.ftlh [hunk 1]
```diff
@@ -1,32 +1,33 @@
...
-@import ore.markdown.MarkdownRenderer
-@import views.html.helper.CSPNonce
-@import views.html.utils
+-->
+<#--@import controllers.sugar.Requests.OreRequest-->
+<#--@import models.viewhelper.{ProjectData, ScopedProjectData}-->
+<#--@import ore.OreConfig-->
+<#--@import ore.markdown.MarkdownRenderer-->
+<#--@import views.html.helper.CSPNonce-->
+<#--@import views.html.utils-->
```
Comment (KennyTV) [inline src/main/resources/templates/projects/discuss.ftlh:14]: Not like this is hurting in this instance, but commenting out the entire block vs. every single line would probably be nicer :p

File: src/main/resources/templates/projects/discuss.ftlh [hunk 2]
```diff
@@ -44,11 +45,12 @@ Discussion page within Project overview.
...
-                    saveCall = routes.getRouteUrl("projects.postDiscussionReply", p.project.ownerName, p.project.slug),
-                    cancellable = false,
-                    enabled = true
-                    )
+                    <@editor.editor saveCall=routes.getRouteUrl("projects.postDiscussionReply", p.project.ownerName, p.project.slug) cancellable=false enabled=true/>
+<#--                    @utils.editor(-->
+<#--                    saveCall = routes.getRouteUrl("projects.postDiscussionReply", p.project.ownerName, p.project.slug),-->
+<#--                    cancellable = false,-->
+<#--                    enabled = true-->
+<#--                    )-->
```
Comment (KennyTV) [inline src/main/resources/templates/projects/discuss.ftlh:53]: Please don't just comment out non-ported blocks (with single line comments), that just makes porting them later *ever so slightly slightly* more annoying

File: src/main/resources/templates/utils/form.ftlh [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<#macro form action method id="" class="" enctype="">
+<#macro form action method="" id="" class="" enctype="">
```
Comment (KennyTV) [inline src/main/resources/templates/utils/form.ftlh:1]: giving this a default prolly just makes us forget to set one properly

File: src/main/resources/templates/users/notifications.ftlh [hunk 4]
```diff
@@ -156,7 +155,7 @@
 
                             <div class="invite-message invite-accepted" style="display: none;">
                                 <i class="minor fas fa-3x fa-birthday-cake"></i><br/>
-                                @Html(messages("notification.invite.joined", subject.name))<br/>
+                                @Html(<@spring.messageArgs "notification.invite.joined"  subject.name />)<br/>
```
Comment (KennyTV) [inline src/main/resources/templates/users/notifications.ftlh:158]: you can remove the outer html wrap in these cases

Comment (MiniDigger) [reply src/main/resources/templates/users/notifications.ftlh:158]: Yes, freemarker doesn't escape HTML, unless you tell it to

File: src/main/resources/templates/projects/discuss.ftlh (hunk 1 above)
Comment (yannicklamprecht) [reply src/main/resources/templates/projects/discuss.ftlh:14]:
```suggestion
<#--@import controllers.sugar.Requests.OreRequest
@import models.viewhelper.{ProjectData, ScopedProjectData}
@import ore.OreConfig
@import ore.markdown.MarkdownRenderer
@import views.html.helper.CSPNonce
@import views.html.utils-->
```

**Comment by Machine-Maker** [comment]
For some reason, this isn't showing the later commits... gonna remake

**Pull request closed without merging**